app/templates/suppliers/already_completed.html

```diff
@@ -0,0 +1,57 @@
+{% extends "_base_page.html" %}
+
+{% block page_title %}Change your {{ completed_data_description }} – Digital Marketplace{% endblock %}
+
+{% block breadcrumb %}
+  {% with items = [
+    {
+      "link": "/",
+      "label": "Digital Marketplace"
+    },
+    {
+       "link": url_for('.dashboard'),
+       "label": "Your account"
+    },
+    {
+      "link": url_for('.supplier_details'),
+      "label": "Company details"
+    },
+  ] %}
+    {% include "toolkit/breadcrumb.html" %}
+  {% endwith %}
+{% endblock %}
+
+{% block main_content %}
+
+<div class="single-question-page">
+  <div class="grid-row">
+
+    <div class="column-two-thirds dmspeak">
```

Comment:
What does this `dmspeak` bring in again?

It's not on the pages I've made.

Reply:
Yes I think it helps with the bullets, also spacing between paragraphs and at the bottom of the section I think.